libs/common-const/src/common.ts

```diff
@@ -52,19 +48,12 @@ export const APP_TITLE = 'CoW Swap | The smartest way to trade cryptocurrencies'
 
 type Env = 'barn' | 'prod'
 
+const COWSWAP_ETHFLOW_CONTRACT_PROD = '0x40A50cf069e992AA4536211B23F286eF88752187'
+const COWSWAP_ETHFLOW_CONTRACT_BARN = '0xD02De8Da0B71E1B59489794F423FaBBa2AdC4d93'
+
 export const COWSWAP_ETHFLOW_CONTRACT_ADDRESS: Record<Env, Partial<Record<SupportedChainId, string>>> = {
-  prod: {
-    [SupportedChainId.MAINNET]: EthFlowProd[SupportedChainId.MAINNET].address,
-    [SupportedChainId.GNOSIS_CHAIN]: EthFlowProd[SupportedChainId.GNOSIS_CHAIN].address,
-    [SupportedChainId.GOERLI]: EthFlowProd[SupportedChainId.GOERLI].address,
-    [SupportedChainId.SEPOLIA]: EthFlowProd[SupportedChainId.SEPOLIA].address,
-  },
-  barn: {
-    [SupportedChainId.MAINNET]: EthFlowBarn[SupportedChainId.MAINNET].address,
-    [SupportedChainId.GNOSIS_CHAIN]: EthFlowBarn[SupportedChainId.GNOSIS_CHAIN].address,
-    [SupportedChainId.GOERLI]: EthFlowBarn[SupportedChainId.GOERLI].address,
-    [SupportedChainId.SEPOLIA]: EthFlowBarn[SupportedChainId.SEPOLIA].address,
-  },
+  prod: mapAddressToSupportedNetworks(COWSWAP_ETHFLOW_CONTRACT_PROD),
+  barn: mapAddressToSupportedNetworks(COWSWAP_ETHFLOW_CONTRACT_BARN),
```

Comment:
Not 100% sure.
Right now the addresses are the same, but it's not guaranteed.
See [what Federico said](https://cowservices.slack.com/archives/C049GSMCTMW/p1671450797834959?thread_ts=1671449420.113069&cid=C049GSMCTMW)

Reply:
Yeah, I understand.
For some reason, locally, yarn wrongly resolves `@cowprotocol/ethflowcontract` package:
https://cowservices.slack.com/archives/C0361CDG8GP/p1704727878010159

I have no idea why it's happening, but I also faced it before and can't reproduce now.
The only way I see to solve it - hardcoding

Reply:
Maybe it has something to do with yarn.lock?

Reply:
As far as I can see it contains proper data